pkg/core/model/message.go

```diff
@@ -1,9 +1,29 @@
+/*
+Copyright 2019 The KubeEdge Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package model is
```

Comment:
please complete the comment

Reply:
ok

Reply:
@sids-b , the comment is :
```
// Package model is the definition of Message.  Message is defined by protobuf. it content Header, Router and Data. 
```

is that ok ?